README.md

```diff
@@ -6,3 +6,22 @@ closed.
 This package is disabled by default and can be enabled via the
 `autosave.enabled` config setting or from the Autosave section of the Settings
 view (`cmd-,`).
+
+## Service API
+
+### package.json
```

Comment:
This example is great, but could you add some text in english describing this as well?